client/state/site-roles/actions.js

```diff
@@ -1,44 +0,0 @@
-/**
- * Internal dependencies
- */
-import wpcom from 'calypso/lib/wp';
-import {
-	SITE_ROLES_RECEIVE,
-	SITE_ROLES_REQUEST,
-	SITE_ROLES_REQUEST_FAILURE,
-	SITE_ROLES_REQUEST_SUCCESS,
-} from 'calypso/state/action-types';
-
-import 'calypso/state/site-roles/init';
-
-export function requestSiteRoles( siteId ) {
-	return ( dispatch ) => {
-		dispatch( {
-			type: SITE_ROLES_REQUEST,
-			siteId,
-		} );
-
-		return wpcom
-			.undocumented()
-			.site( siteId )
-			.getRoles()
```

Comment:
Seems like we can also delete this method from `wpcom.undocumented().site()`?

Reply:
good catch, fixed here: https://github.com/Automattic/wp-calypso/pull/53503/commits/9a60081c98dc47c4056119a51b7db8ef0f2bde3c 👍🏻